### PR TITLE
Update italian translation

### PR DESCRIPTION
**Pull request by bovirus**

### CODE REVIEW

File: recovery/translation_it.ts [hunk 1]
```diff
@@ -1767,7 +1767,7 @@ Questo è molto utile quando si usa una connessione VNC.</translation>
     <message>
         <location filename="optionsdialog.ui" line="438"/>
         <source> When used in conjunction with forcetrigger it sets a maximum timeout period in seconds for the recovery menu to be displayed before continuing to the boot selection dialog and potentially selecting a default OS to boot.</source>
```
Comment (lurch) [inline recovery/translation_it.ts:1769]: Is the space at the start of the string necessary, @procount ?

**Comment by bovirus** [comment]
@procount 

Could you please merge this update for Italian language and close the PR?

**Comment by procount** [comment]
Please check latest commit.

**Comment by procount** [comment]
Hopefully this is all merged and fixed now.

**Pull request closed without merging**